packages/cfd/src/types/cfd-store.types.ts

```diff
@@ -1,21 +1,36 @@
 import { DetailsOfEachMT5Loginid, Mt5NewAccount, VerifyEmailResponse } from '@deriv/api-types';
-import { TTradingPlatformAvailableAccount } from 'Components/props.types';
+import { TShortcode, TTradingPlatformAvailableAccount } from 'Components/props.types';
 import { TCFDPasswordFormValues } from 'Containers/cfd-password-modal';
 import { TDerivezCompanies, TDxCompanies, TMtCompanies } from 'Stores/Modules/CFD/Helpers/cfd-config';
 import { FormikHelpers } from 'formik';
 
-type TStoreProofOfAddressArgs = {
+export type TStoreProofOfAddressArgs = {
     file_uploader_ref: HTMLDivElement | null;
     values: {
         [key: string]: string;
     };
 };
 
+export type TAccountType = {
+    category: 'real' | 'demo' | '';
+    type?:
+        | 'all'
```

Comment:
Do not use `string` type with literal types.

Reply:
We have to refactor the code as already mentioned that account_type type is being used in so many places in cfd package and it requires more effort and time. We need separate cards for this and it can be removed gradually not in one go